python/MooseDocs/extensions/appsyntax.py

```diff
@@ -105,6 +105,8 @@ def defaultConfig():
         config['alias'] = (None, "Dictionary of syntax aliases.")
         config['unregister'] = (None,
                                 "A `dict` or `dict` of `dict` including syntax to unregister (key='moose_base', value='parent_syntax')")
+        config['markdown'] = (None,
+                             "A `dict` or `dict` of `dict` including markdown files to explicitly set markdown reference (key='syntax', value='file.md')")
```

Comment:
```suggestion
        config['markdown'] = (None,
                             "A `dict` or `dict` of `dict` including markdown files to explicitly set a custom, additional markdown reference (key='syntax', value='file.md')")
```

Would like to make it clear in the description for developers that this is custom and in addition to the tree we already build.